pkgs/by-name/pi/pianobar/package.nix

```
@@ -6,7 +6,7 @@
   libao,
   json_c,
   libgcrypt,
-  ffmpeg,
+  ffmpeg_7,
```

```suggestion
  ffmpeg,
```
not needed: https://github.com/NixOS/nixpkgs/pull/462282